### PR TITLE
Add Course

### PR DESCRIPTION
**Pull request by naremanhilles**

### CODE REVIEW

**Comment by kannanhassouna** [comment]
Description ?!!

**Comment by AsalaKM** [comment]
@naremanhilles  Please!
- Write a clear description for your work
- It can't be in progress and need review in the same time

**Comment by kannanhassouna** [comment]
@naremanhilles , Good job btw. 
but you have a lot of repeated code you should take care of that

File: Client/src/components/Layout/Desktop/index.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: #eee;
+}
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.css:3]: Don't use the body tag in css 
add a class and use it

File: Client/src/components/Layout/Desktop/index.js [hunk 2]
```diff
@@ -1,34 +1,33 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.js:1]: use single quotation 
```suggestion
import React, { Component } from 'react';
```

File: Client/src/components/Layout/Desktop/index.js [hunk 3]
```diff
@@ -1,34 +1,33 @@
...
 import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.js:2]: use single quotation
```suggestion
import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
```

File: Client/src/components/Layout/Desktop/index.js [hunk 4]
```diff
@@ -1,34 +1,33 @@
...
-import Footer from '../DesktopFooter';
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.js:3]: add a new line after import third module

File: Client/src/components/Layout/Desktop/index.js [hunk 5]
```diff
@@ -1,34 +1,33 @@
...
   render() {
     const { login } = this.state;
     return (
-      <BrowserRouter className='desktop-container'>
+      <BrowserRouter className="desktop-container">
         {login ? (
-          <>
-            <SideBar />
+          <div className="s">
+            <Sidebar />
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.js:20]: className s ?? 
use readable names

File: Client/src/components/Layout/Desktop/index.js [hunk 6]
```diff
@@ -1,34 +1,33 @@
...
             <Switch>
-              <Route exact path={'/Courses'} component={Courses} />
-              <Route exact path={'/Student'} component={Student} />
+              <Route exact path={"/courses"} component={Courses} />
+              <Route exact path={"/Student"} component={Student} />
             </Switch>
```
Comment (AbdallahAmmar96) [inline Client/src/components/Layout/Desktop/index.js:24]: use single quotation

**Review by AbdallahAmmar96 — CHANGES_REQUESTED** [review]

File: Client/src/components/Layout/Desktop/index.js (hunk 2 above)
Comment (AsalaKM) [reply Client/src/components/Layout/Desktop/index.js:1]: @naremanhilles  check your eslint

File: Client/src/components/contanier/staff/Courses/AddCourse/index.css [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+  display: flex;
+}
+.add-course-titel {
+  color: #06468f;
+  letter-spacing: 1px;
+  font-size: 34px;
+  font-weight: bold;
+  margin-top: 50px;
+  margin-left: 39px;
+  margin-bottom: 9px;
```
Comment (AsalaKM) [inline Client/src/components/contanier/staff/Courses/AddCourse/index.css:13]: In all your work
```
margin-top: 50px;
  margin-left: 39px;
  margin-bottom: 9px;
```
replace it with 
```
margin:;
```
use one property to reduce your lines

File: Client/src/components/contanier/staff/Courses/AddCourse/index.css [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+}
+.start-label {
+  display: flex;
+  flex-direction: column;
+  color: #06468f;
+  font-size: 14px;
+  margin-bottom: 7px;
+  margin-right: 16px;
+}
+.start-input {
```
Comment (AsalaKM) [inline Client/src/components/contanier/staff/Courses/AddCourse/index.css:119]: `.start-input` and `.end-input`
`..start-label ` and `.end-label `
the same style for both !!
use one class for it 
don't repeat your code :+1:

File: Client/src/components/contanier/staff/Courses/AddCourse/index.css [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+  color: white;
+  background-color: #2980b9;
+  border-radius: 5px;
+  outline: none;
+  font-size: 14px;
+  margin-top: 22px;
+  margin-left: 18px;
+  border: none;
+}
+.add-button {
```
Comment (AsalaKM) [inline Client/src/components/contanier/staff/Courses/AddCourse/index.css:156]: remove it

**Review by AsalaKM — CHANGES_REQUESTED** [review]

**Pull request closed without merging**